sklearn/metrics/classification.py

```diff
@@ -313,8 +315,11 @@ class labels [2].
     .. [1] J. Cohen (1960). "A coefficient of agreement for nominal scales".
            Educational and Psychological Measurement 20(1):37-46.
            doi:10.1177/001316446002000104.
-    .. [2] R. Artstein and M. Poesio (2008). "Inter-coder agreement for
+    .. [2] `R. Artstein and M. Poesio (2008). "Inter-coder agreement for
```

Comment:
I've found examples ([here](https://github.com/hlin117/scikit-learn/blob/8e0fcdf6898e2723987d6f45268c631b09e3e32b/sklearn/metrics/classification.py#L1677) and [here](https://github.com/hlin117/scikit-learn/blob/8e0fcdf6898e2723987d6f45268c631b09e3e32b/sklearn/metrics/classification.py#L969)) where the entire reference is linked, and not just the title. 

Do you still want me to only link the titles?